pnpm-lock.yaml

```diff
@@ -1533,6 +1533,10 @@ packages:
     resolution: {integrity: sha512-FtwrG/euBzaEjYeRqOgly7G0qviiXoJWnvEH2Z1plBdXgbyjv34pHTSb9zoeHMyDy33+DWy5Wt9Wo+TURtOYSQ==}
     engines: {node: '>= 0.8'}
 
+  http-errors@2.0.1:
```

Comment:
needs deduplication

Reply:
can't be deduplicated because `raw-body` (from express) has fixed dependency to 2.0.0